pkgs/development/python-modules/backports_shutil_which/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, fetchPypi, buildPythonPackage }:
+
+buildPythonPackage rec {
+  pname = "backports.shutil_which";
+  version = "3.5.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "16sa3adkf71862cb9pk747pw80a2f1v5m915ijb4fgj309xrlhyx";
+  };
+
+  # Tests fail: "ValueError: underlying buffer has been detached"
```

Comment:
Is this comment copy-and-pasted from python-u2flib-host?

Reply:
I tried to build and the test phase raised that error. I tried to fix it but it has probably do something with the test environment in nix. Not sure though.

Reply:
For me this test works perfectly fine.